### ISSUE TITLE
Add FAQ Section

### ISSUE THREAD

**Issue by danielsimao** [bob-collective/bob-ui#16]
### Description

Add FAQ Section

**Comment by danielsimao** [comment]
@nud3l @tomjeatt  do we have the text for the faq section?

**Comment by tomjeatt** [comment]
> @nud3l @tomjeatt do we have the text for the faq section?

Nope, not yet—let's pick this up next week. I'll add it to the task list. We'll probably need to speak to Zhixi/Chris, and find out what questions they're being asked.
